rollup.config.js

```diff
@@ -0,0 +1,16 @@
+import replace from 'rollup-plugin-replace';
+
+export default {
+  input: 'src/index.js',
+  output: {
+    file: 'dist/hyperapp.js',
+    format: 'umd',
+    name: 'hyperapp',
+    sourcemap: true
+  },
+  plugins: [
+    replace({
+      ENVIRONMENT: JSON.stringify(process.env.NODE_ENV || 'production')
```

Comment:
Why not use the same `NODE_ENV` for the identifier to replace?

Reply:
Done

Reply:
Thanks, this is what I've done on other projects that needed to branch based on build environment and I think it will probably be more obvious where values come from.